tests/HUBViewControllerTests.m

```diff
@@ -1511,6 +1511,29 @@ - (void)testSetScrollOffsetForwardsAnimatedFlagToCollectionView
     XCTAssertTrue(self.collectionView.appliedScrollViewOffsetAnimatedFlag);
 }
 
+- (void)testRenderingUpdatesContentInsetBeforeAndAfterRendering
+{
+    UIEdgeInsets const firstInsets = UIEdgeInsetsMake(100, 30, 40, 200);
+    UIEdgeInsets const secondInsets = UIEdgeInsetsMake(50, 0, 0, 0);
+    __block NSUInteger calls = 0;
+    
+    __weak HUBViewControllerTests *weakSelf = self;
+    self.scrollHandler.contentInsetHandler = ^UIEdgeInsets(UIViewController<HUBViewController> *controller, UIEdgeInsets insets) {
+        HUBViewControllerTests *strongSelf = weakSelf;
+        if (++calls == 1) {
```

Comment:
I can see why they removed these operators from Swift now 😅 This is a bit hard to read, could you make the increment outside of the `if` statement instead? 😇